Bridges/HttpKernel.php

```diff
@@ -240,12 +238,22 @@ protected function mapResponse(HttpResponse $reactResponse, SymfonyResponse $syR
 
         $reactResponse->writeHead($syResponse->getStatusCode(), $headers);
 
-        $stdOut = '';
-        while ($buffer = @ob_get_clean()) {
-            $stdOut .= $buffer;
+        // asynchronously get content
+        ob_start(function($buffer) use ($reactResponse) {
+            $reactResponse->write($buffer);
```

Comment:
Since all stuff here is asycnhronouse we can not do async stuff here. That will break since reactResponse->end() is called immediately some lines below. Also I don't get the point of using async method here because between this line and $reactResponse->end is nothing that can do something async.